distributed/tests/test_scheduler.py

```diff
@@ -717,19 +717,20 @@ async def test_server_listens_to_other_ops(s, a, b):
         assert ident["id"].lower().startswith("scheduler")
 
 
-@gen_cluster()
-async def test_remove_worker_from_scheduler(s, a, b):
-    dsk = {("x-%d" % i): (inc, i) for i in range(20)}
-    s.update_graph(
-        tasks=valmap(dumps_task, dsk),
-        keys=list(dsk),
-        dependencies={k: set() for k in dsk},
-    )
```

Comment:
Most update_graph invocations where very old tests that were written at a time where the client was not what it is today. I migrated to a more modern version of these tests but kept functionality as is.